torch/testing/_internal/common_modules.py

```diff
@@ -440,8 +492,7 @@ def module_inputs_torch_nn_MultiheadAttention(module_info, device, dtype, requir
                module_inputs_func=module_inputs_torch_nn_Hardswish,
                supports_gradgrad=False),
     ModuleInfo(torch.nn.TransformerEncoderLayer,
-               module_inputs_func=module_inputs_torch_nn_TransformerEncoderLayer,
-               supports_gradgrad=False),
+               module_inputs_func=module_inputs_torch_nn_TransformerEncoderLayer),
```

Comment:
Was `supports_gradgrad=False` intentionally removed?

Reply:
Yeah. I didn't see any comment mentioning why it was added in https://github.com/pytorch/pytorch/pull/64444 and the test worked so.